ci/cloudbuild/builds/integration-daily.sh

```diff
@@ -46,6 +46,12 @@ export GOOGLE_CLOUD_CPP_IAM_QUOTA_LIMITED_INTEGRATION_TESTS="yes"
 mapfile -t integration_args < <(integration::bazel_args)
 integration::bazel_with_emulators test "${args[@]}" "${integration_args[@]}"
 
+io::log_h2 "Running Bigtable integration tests (against prod)"
+bazel test "${args[@]}" "${integration_args[@]}" \
+  --test_tag_filters="integration-test" -- \
+  "//google/cloud/bigtable/..." \
```

Comment:
If we're going to be absolute, you might as well add the `//` to the Spanner target below while we're here.

Reply:
Done.

Reply:
Thanks.